src/View.php

```diff
@@ -516,7 +526,7 @@ public function on($event, $selector = null, $action = null)
             // if callable $action is passed, then execute ajaxec()
             //
             throw new Exception('VirtualPage is not yet implemented');
-            $url = '.virtualpage->getURL..';
+            /*$url = '.virtualpage->getURL..';
```

Comment:
yah, it's just blocking unfinished code, so it's intended. That area is not reachable anyway.